src/Cake.Common/Tools/InnoSetup/InnoSetupSettings.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using Cake.Core.IO;
+using Cake.Core.Tooling;
+
+namespace Cake.Common.Tools.InnoSetup
+{
+    /// <summary>
+    /// Contains settings used by the <see cref="InnoSetupRunner"/>.
+    /// </summary>
+    public sealed class InnoSetupSettings : ToolSettings
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="InnoSetupSettings"/> class with the default settings.
+        /// </summary>
+        public InnoSetupSettings()
```

Comment:
@thomaslevesque I don't have the solution open, but it would likely be this rule: https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1201.md